src/components/ProblemsTable.tsx

```diff
@@ -234,22 +314,29 @@ export const ProblemsTable = ({
           Header: displayStrings.fileName,
           Filter: tableFilters.TextFilter(),
           minWidth: 150,
-          cellRenderer: ({ cellElementProps, cellProps }: CellRendererProps<TableRow>) => {
+          Cell: (cellProps: CellProps<TableRow & SourceFile>) => {
             const extension = cellProps.value?.substring(cellProps.value.lastIndexOf('.') + 1);
-            return (
-              <DefaultCell
-                cellElementProps={cellElementProps}
-                cellProps={cellProps}
-                startIcon={
-                  extension && extension in filetypeIcons ? (
+
+            return cellProps.row.depth != 0 &&
+              context?.currentTable &&
+              tableStyleAccessor[context?.currentTable] === 'fileId' ? (
```

Comment:
same comment about currentTable type, find all spots we hard check the table type and not check the defined table types